src/NUnitConsole/nunit4-console/ConsoleRunner.cs

```diff
@@ -324,8 +325,8 @@ private void DisplayExtensionList()
                 {
                     _outWriter.Write(INDENT6 + "Extension: ");
                     _outWriter.Write(ColorStyle.Value, $"{node.TypeName}");
-                    if(node.TargetFramework != null)
-                        _outWriter.Write(ColorStyle.Value, $"(.NET {node.TargetFramework?.FrameworkVersion})");
+                    //if(node.TargetFramework != null)
+                    //    _outWriter.Write(ColorStyle.Value, $"(.NET {node.TargetFramework?.FrameworkVersion})");
```

Comment:
Remove.

Reply:
Yup. I was hoping to restore it but no dice for now. Maybe after all the changes are done and we have replaced RuntimeFramework with something more modern.